tesseract/tesseract_planning/tesseract_motion_planners/src/trajopt/problem_generators/default_problem_generator.cpp

```diff
@@ -402,7 +402,10 @@ trajopt::TrajOptProb::Ptr DefaultTrajoptProblemGenerator(const std::string& name
   TrajOptCompositeProfile::Ptr cur_composite_profile{ nullptr };
   auto it_composite = composite_profiles.find(profile);
   if (it_composite == composite_profiles.end())
+  {
+    CONSOLE_BRIDGE_logDebug("Trajopt profile not found. Setting default");
     cur_composite_profile = std::make_shared<TrajOptDefaultCompositeProfile>();
```

Comment:
Did we want to switch this to trying to find a "DEFAULT" profile?

Reply:
Yeah I would like to. I didn't intend to do that in this PR though. I can remove this

Reply:
Sounds good. This is fine for now.